tasks.py

```diff
@@ -383,20 +383,22 @@ def create_monetdb(
         run(c, cmd)
 
 
-@task(iterable=["port"])
-def init_monetdb(c, port):
+@task(iterable=["worker"])
```

Comment:
Great work! 

I think we need to update the documentation to state that the globalnode need the metadata as well now to be loaded.